WORKSPACE

```diff
@@ -36,11 +36,11 @@ py_repositories()
 
 http_archive(
     name = "io_bazel_rules_closure",
-    sha256 = "9b99615f73aa574a2947226c6034a6f7c319e1e42905abc4dc30ddbbb16f4a31",
-    strip_prefix = "rules_closure-4a79cc6e6eea5e272fe615db7c98beb8cf8e7eb5",
+    sha256 = "6a900831c1eb8dbfc9d6879b5820fd614d4ea1db180eb5ff8aedcb75ee747c1f",
```

Comment:
Are these changes required? If not, please drop them.

Reply:
Sorry, I hate Git.  It only showed the one file in the PR, else I would have removed this before.